contrib/format-spss/README.md

```diff
@@ -0,0 +1,83 @@
+# Format Plugin for SPSS (SAV) Files
+This format plugin enables Apache Drill to read and query Statistical Package for the Social Sciences (SPSS) (or Statistical Product and Service Solutions) data files. According
+ to Wikipedia: (https://en.wikipedia.org/wiki/SPSS)
+ ***
+ SPSS is a widely used program for statistical analysis in social science. It is also used by market researchers, health researchers, survey companies, government, education researchers, marketing organizations, data miners, and others. The original SPSS manual (Nie, Bent & Hull, 1970) has been described as one of "sociology's most influential books" for allowing ordinary researchers to do their own statistical analysis. In addition to statistical analysis, data management (case selection, file reshaping, creating derived data) and data documentation (a metadata dictionary is stored in the datafile) are features of the base software.
+ ***
+ 
+## Configuration 
+To configure Drill to read SPSS files, simply add the following code to the formats section of your file-based storage plugin.  This should happen automatically for the default
```

Comment:
Do we want to add all the format plugins at bootstrap? Creates a rather intimidating-looking hunk of JSON for newbies. Of course, it would be good for format plugins to be independent of the storage plugin, but that will come later.

Reply:
My thought is that it's better to add it in the bootstrap so that people know it's there.  Just my .02...